README.md

```diff
@@ -42,7 +45,8 @@ On CoreOS because /etc/*-release are symlinks you want to mount
 
```

Comment:
Seems like that sentence above is messing with the italics parsing in Markdown. Consider putting the paths in monospace font with \` : `/etc/*-release`, etc